### ISSUE TITLE
training.txt contains images in testing.txt

### ISSUE THREAD

**Issue by leighton613** [xiaohaoChen/rrc_detection#14]
Hi! Thanks for your code and paper.

Wonder why the `testing.txt` and `training.txt` are different from the description in the paper which says validation has 2741 images.

In the repo, `training.txt` contains all the car images (6684), and the `testing.txt` has a subset (503). How did you pick this subset? Should I remove them from the training set to get validation results?

Thanks ahead!

**Comment by xiaohaoChen** [comment]
Hi!
The `testing.txt` was just set for running the script. Since the `training.txt` contains all the car images, we didn't test in the validation set while training. The `testing.txt` is invalid here. 
If you need to do some experiments with validation set, I can sent you the `testing.txt` and `training.txt` as we descripted in the paper.

**Comment by leighton613** [comment]
Hi Xiaohao,
Thanks so much for sharing! Here's my email: leighton.zxw@gmail.com ;)

**Comment by ChenLi2013** [comment]
Hi XiaohaoChen,
          Could you please send the testing.txt and training.txt  of  car ,pedestrian,cyclist  to me?
          Here's my email: lilichen0906@163.com ;

**Comment by griffintin** [comment]
@xiaohaoChen  could you please send me the training.txt and testing.txt ?  The address is Guifen.Tian@gmail.com, thank you in advance

**Comment by griffintin** [comment]
@xiaohaoChen   Could you please send me the training.txt and testing.txt ? The address is tianguifen@gmail.com.  Thank you so much !

**Comment by foralliance** [comment]
Hi XiaohaoChen,
Could you please send the testing.txt and training.txt of car ,pedestrian,cyclist to me?
Here's my email: [nedved-f@163.com](url) 
many many thanks

**Comment by foralliance** [comment]
@xiaohaoChen 

`'test_interval': 1000000,`

Because of this setting, there is no test in the training process.
That's what you said earlier."**we didn't test in the validation set while training**"???

**Comment by foralliance** [comment]
@leighton613 @xiaohaoChen @ChenLi2013 Hi 
Could you please send the testing.txt and training.txt of car ,pedestrian,cyclist to me?
Here's my email: nedved-f@163.com
many many thanks

**Comment by hulaifeng** [comment]
@xiaohaoChen @ChenLi2013 @leighton613 @foralliance 
Could you please send the testing.txt and training.txt of car ,pedestrian,cyclist to me?
 Here's my email: 201221100501@std.uestc.edu.cn
thanks
